File: Codeforces/Round-973/a.py
# Author: Angel Quintana
# Mexico City
import math
def run():
    n = int(input())
    x, y = map(int, input().split())

    p = math.ceil(n/min(x,y))
    print(p)
if __name__ == "__main__":
    tc = int(input())
    # tc = 1
    for i in range(tc):
        run()

File: Codeforces/Round-973/A.cpp
// Author: Angel Quintana

#include <bits/stdc++.h>

using namespace std;

void solve(){
  int n, x, y;
  cin >> n >> x >> y;
  
  int c = 0;
  while(n > 0){
    int aux = min(x,y);
    n-= aux;
    c++;
  }
  cout << c << endl;
}

int main(){
    ios_base::sync_with_stdio(0); 
    cin.tie(0);

    int tt;
    cin >> tt;
    //tt = 1;
    for(int i=0; i<tt; ++i){
        solve();
    }

    return 0;
}
